airbyte-integrations/connectors/source-close-com/source_close_com/single_stream_slicer.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+import pendulum
+
+from dataclasses import dataclass, field
+from typing import Optional, Iterable, Mapping, Any, Union
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.datetime import MinMaxDatetime
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.request_option import RequestOptionType, RequestOption
+from airbyte_cdk.sources.declarative.stream_slicers import SingleSlice
+from airbyte_cdk.sources.declarative.types import StreamState, StreamSlice, Record, Config
+
+
+@dataclass
+class SingleStreamSlicer(SingleSlice):
```

Comment:
what is the need to implement a new class here? regarding you're extending the `SingleSlice` stream, do you really need overriding this vast amount of data class attributes and methods?

Reply:
?

Reply:
Not sure I got your question, sorry

Reply:
nevermind, I've answered the question myself